client/packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -92,56 +38,62 @@ const SyncSettingsForm = ({
   isDisabled: boolean;
   isSaving: boolean;
   isValid: boolean;
-  settings?: SyncSettings;
+  settings: SyncSettingsInput;
   onSave: () => void;
-  setSyncSettings: (syncSettings: SyncSettings) => void;
+  setSyncSettings: (syncSettings: SyncSettingsInput) => void;
 }) => {
   const t = useTranslation('common');
+  const setSettings = (
+    property: keyof SyncSettingsInput,
+    value: number | string
+  ) => setSyncSettings({ ...settings, [property]: value });
+
+  const { url, username, password, intervalSeconds } = settings;
   return (
     <form style={{ width: '100%' }}>
       <Setting
         title={t('label.settings-url')}
         component={
-          <StringSyncSetting
-            property="url"
-            settings={settings}
+          <BasicTextInput
+            value={url}
+            onChange={e => setSettings('url', e.target.value)}
+            type={'text'}
             disabled={isDisabled}
-            update={setSyncSettings}
           />
         }
       />
       <Setting
         title={t('label.settings-username')}
         component={
-          <StringSyncSetting
-            property="username"
-            settings={settings}
+          <BasicTextInput
+            value={username}
+            onChange={e => setSettings('username', e.target.value)}
+            type={'text'}
             disabled={isDisabled}
-            update={setSyncSettings}
           />
         }
       />
       <Setting
         title={t('label.settings-password')}
         component={
-          <StringSyncSetting
-            property="password"
-            type="password"
-            autocomplete="sync-password"
-            settings={settings}
+          <BasicTextInput
```

Comment:
Would be nice to use the new password input after the merging happens ( either develop into this branch, or this one into develop )

Reply:
I've updated now from develop, and changed to password field, thanks (this is for head pr)